image_provisioner/playbooks/roles/docker-config/templates/etc/sysconfig/docker-storage-setup.j2

```diff
@@ -9,7 +9,7 @@ DATA_SIZE=100%FREE
 {% elif docker_storage_driver == 'overlay2' %}
 {% if atomic | bool %}
 CONTAINER_ROOT_LV_NAME=docker-root-lv
-CONTAINER_ROOT_LV_MOUNT_PATH=/var/lib/docker
+CONTAINER_ROOT_LV_MOUNT_PATH=/var/lib
```

Comment:
What takes care of mounting an overlay volume at /var/lib?

Reply:
docker when started starts the docker-storage-setup which takes care of it.

Reply:
Why are we making all of `/var/lib` into an overlay2 lv? Docker storage setup should be for docker only AFAIK, CRI-O does its own thing to set up `/var/lib/containers`.

Reply:
docker storage setup will assign all of the free space to /var/lib/docker, so /var/lib/containers will not have much storage. Making all of /var/lib in to an overlay2 lv will solve the issue.